packages/interface/src/components/fullscreen-mode/style.scss

```diff
@@ -10,14 +9,80 @@ body.js.is-fullscreen-mode {
 			height: calc(100% + #{ $admin-bar-height });
 		}
 
-		#adminmenumain,
+		// Hide the top adminbar toolbar.
 		#wpadminbar {
 			display: none;
 		}
 
+		// Display the sidebar offscreen, and then transition in when needed.
+		#adminmenumain {
+			position: relative;
+			top: 60px;
+			left: -160px;
+			transition: left 0.12s ease-in-out;
+		}
+
+		// Hides the admin menu
+		// This prevents keyboard navigation from accessing inner links
+		#adminmenuwrap {
+			display: none;
+		}
+
+		#adminmenu {
+			position: relative;
+			bottom: auto;
+		}
+
+		#adminmenuback {
+			top: 60px;
+		}
+
+		.edit-post-header-toolbar__block-toolbar,
```

Comment:
We shouldn't be targeting edit-post styles on the interface package. What is this about?

Reply:
Looks like there's more of these styles a bit under. can't we target the skeleton header instead?

Reply:
I see. That makes sense. I've updated it.